targets/PythonSdk/make.js

```diff
@@ -0,0 +1,407 @@
+var path = require("path");
+
+// Making resharper less noisy - These are defined in Generate.js
+if (typeof (templatizeTree) === "undefined") templatizeTree = function () { };
+if (typeof (getCompiledTemplate) === "undefined") getCompiledTemplate = function () { };
+
+// generate.js looks for some specific exported functions in make.js, like:
+exports.makeClientAPI2 = function (apis, sourceDir, apiOutputDir) {
+    var locals = {
+        apis: apis,
+        buildIdentifier: exports.buildIdentifier,
+        errorList: apis[0].errorList,
+        errors: apis[0].errors,
+        friendlyName: "PlayFab Python Client Sdk",
+        sdkVersion: exports.sdkVersion
+    };
+
+    // Builds the client api.  The provided "api" variable is a single object, the API_SPECS/client.api.json as an object
+    console.log("Generating Client api from: " + sourceDir + " to: " + apiOutputDir);
+
+    templatizeTree(locals, path.resolve(sourceDir, "source"), apiOutputDir);
+
+    for (var i = 0; i < apis.length; i++)
+        makeApi(apis[i], sourceDir, apiOutputDir);
+    generateSimpleFiles(apis, sourceDir, apiOutputDir);
+}
+
+
+// generate.js looks for some specific exported functions in make.js, like:
+exports.makeServerAPI = function (apis, sourceDir, apiOutputDir) {
+    var locals = {
+        apis: apis,
+        buildIdentifier: exports.buildIdentifier,
+        errorList: apis[0].errorList,
+        errors: apis[0].errors,
+        friendlyName: "PlayFab Python Server Sdk",
+        sdkVersion: exports.sdkVersion
+    };
+
+    // Builds the server api.  The provided "apis" variable is a list of objects, built from: API_SPECS/admin.api.json, API_SPECS/matchmaker.api.json, and API_SPECS/server.api.json
+    // If you don't want admin, you should filter it out yourself (for now)
```

Comment:
Go ahead and remove this comment. It's copy paste from newtarget.
KUDOS on using NewTarget though, that's great.